subsys/debug/gdbstub/gdbstub_serial.c

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2020 Intel Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <kernel.h>
+#include <drivers/uart.h>
+#include <sys/__assert.h>
+
+static struct device *uart_dev;
+
+int z_gdb_backend_start(void)
+{
+	int ret = 0;
+	static const struct uart_config uart_cfg = {
+		.baudrate = 115200,
+		.parity = UART_CFG_PARITY_NONE,
+		.stop_bits = UART_CFG_STOP_BITS_1,
+		.data_bits = UART_CFG_DATA_BITS_8,
+		.flow_ctrl = UART_CFG_FLOW_CTRL_NONE
+	};
+
+	if (uart_dev == NULL) {
+#ifndef CONFIG_GDBSTUB_SERIAL_BACKEND_NAME
```

Comment:
here, use the default from Kconfig instead of hardcoding uart0...

Reply:
what you mean ? if GDBSTUB_SERIAL_BACKEND_NAME is defined it is used otherwise I just get uart0, is there a symbol to get default serial name ?

Reply:
make GDBSTUB_SERIAL_BACKEND_NAME point to UART0 by default and avoid the conditional here.

Reply:
ow, I see your point, indeed, much better.